tests/content/util.js

```diff
@@ -244,6 +244,18 @@ const PROCESS_YT_VIDEO_TITLE_DATA = [{
 	description: 'should remove 【genre】 from the beginning of the title',
 	args: ['【Genre】 Artist - Track'],
 	expected: { artist: 'Artist', track: 'Track' },
+}, {
+	description: 'should remove CD track number from the beginning of the title',
+	args: [' 01. Artist - Track'],
```

Comment:
Is the leading space added intentionally?

Reply:
Yes it was intentional, I'm adding more tests so I will cover space and no space